src/edges.jl

```diff
@@ -66,7 +66,8 @@ Return `true` if all vertices in `nodes` are connected to each other in the grap
 """
 function isclique(g, nodes)
     for (u, v) in allpairs(nodes)
-        if !isadjacent(g, u, v)
+        #if !isadjacent(g, u, v)
+        if !has_both(g, u, v)
```

Comment:
Chickering is not clear about this but this is how I think it should be. @RobertGregg

Reply:
You were right, Chickering, "Learning equivalence classes..." (2002) defines it.

Reply:
Arrgh, but then this!

<img width="1136" alt="Screenshot 2023-07-23 at 21 31 36" src="https://github.com/mschauer/CausalInference.jl/assets/1923437/a5046d5b-7c17-4009-acce-130876632840">

Reply:
Finally, the solution: 

<img width="1136" alt="Screenshot 2023-07-23 at 21 42 03" src="https://github.com/mschauer/CausalInference.jl/assets/1923437/a4d05c40-e8cd-4f94-9586-3aebf382856e">

Reply:
Okay, that's something 👍

I think I have to read both the Chickering JMLR papers from 2002 properly before I can be of much help regarding such details :/

Reply:
This is from Chickering "Optimal Structure Identification..." 2002 under the lemma 40 proof. I don't think I would have ever spotted this.

![chickering_lemma40 - Copy](https://github.com/mschauer/CausalInference.jl/assets/44711193/e70b1968-e5c6-4ab6-860c-91d4ad1666fd)

I'm confused now at what happens when there is a clique but one of the edges is directed. Do you count it as not being clique? Looks like I'll have to read through this again as well...

Reply:
Ahhh, with neighbors of a node Chickering means nodes connected to it by an undirected edge 😅 (sorry, I was a bit out of the loop)

Robert, if you have a directed edge A -> B in a clique (but the clique does not even matter here) and A and B are both neighbors of a node X (that is, we have X - A and X - B), then this would contradict Lemma 20 posted by Moritz above. (Because X - A and X - B are reversible (aka undirected) and this implies the same for A -> B). 

One detail, I think Lemma 20 should say "in a *completed* PDAG P". 

But I think we got this now 🥳 (this also means that it shouldn't matter if you check for adjacency or undirected edges, as there can only be undirected edges)